### ISSUE TITLE
Update to 0.6.0 failing when trying to load rspec/version

### ISSUE THREAD

**Issue by tarragon** [splitwise/super_diff#120]
Hi,

We had an automated update from 0.5.3 to 0.6.0 fail in CI with the following:

```
Failure/Error: require 'super_diff/rspec-rails'

LoadError:
  cannot load such file -- rspec/version
```

RSpec and related is at the following versions:

```
    rspec-core (3.10.1)
    rspec-expectations (3.10.1)
    rspec-its (1.3.0)
    rspec-mocks (3.10.2)
    rspec-rails (4.0.2)
    rspec-sorbet (1.8.0)
    rspec-support (3.10.2)
```

`require 'rspec/rails'` is earlier in our spec setup.

Adding in a `pry` before the `require 'super_diff/rspec-rails'` and confirmed that trying to manually load `rspec/version` also fails.

Any ideas?

Cheers.

**Comment by mcmire** [comment]
Hmm. My guess is that it moved around in RSpec 3.10. Curious why the integration tests on this side didn't fail. I'll do some digging around.

**Comment by jmartin-sul** [comment]
We also ran into this issue when doing our weekly dependency updates:  https://github.com/sul-dlss/dor-services-app/issues/2131

**Comment by mcmire** [comment]
@tarragon, @jmartin-sul, and everyone who's reacted to this — so sorry about this! I just merged a potential fix, would you mind adjusting your Gemfile to point to `super_diff` master temporarily and see if that works for you?

**Comment by tarragon** [comment]
Tested against master (`d33a157`) and it worked, cheers! 👍🏻

**Comment by jmartin-sul** [comment]
> @tarragon, @jmartin-sul, and everyone who's reacted to this — so sorry about this! I just merged a potential fix, would you mind adjusting your Gemfile to point to `super_diff` master temporarily and see if that works for you?

also tested against the same commit on master, and that worked for us.  thanks!

**Comment by mcmire** [comment]
Great — I just released 0.6.1 with this fix, so check that out!
